src/org/labkey/test/components/ui/search/ManageSampleFinderViewsModal.java

```diff
@@ -126,7 +151,7 @@ public void clickDone()
 
     public String getErrorMsg()
     {
-        if(WebDriverWrapper.waitFor(()->elementCache().errorMsg.isDisplayed(), 1_000))
+        if(Boolean.TRUE.equals(WebDriverWrapper.waitFor(()->elementCache().errorMsg.isDisplayed(), 1_000)))
```

Comment:
Just for my information, is there a difference between primitive 'true' and Boolean.TRUE? 
(this is my paranoia that it's subtly, terribly different talking 🤣 )

Reply:
There is a subtle difference.
`Boolean b = null;` is valid, whereas `boolean b = null;` is not.
This change would only matter if the `WebDriverWrapper.waitFor` could return `null`. The old implementation would throw an NPE if it did but. `WebElement.isDisplayed` can't be null though, so that isn't a concern.